src/transaction/components/TransactionProvider.tsx

```diff
@@ -136,32 +107,17 @@ export function TransactionProvider({
   });
 
   // Transaction Status
-  // For batched, use statusSendCalls or statusWriteContracts
-  // For single, use statusSendCall or statusWriteContract
+  // For batched, use statusSendCalls
+  // For single, use statusSendCall
   const transactionStatus = useMemo(() => {
-    const transactionStatuses = walletCapabilities[Capabilities.AtomicBatch]
-      ?.supported
-      ? {
-          [TRANSACTION_TYPE_CALLS]: statusSendCalls,
-          [TRANSACTION_TYPE_CONTRACTS]: statusWriteContracts,
-        }
-      : {
-          [TRANSACTION_TYPE_CALLS]: statusSendCall,
-          [TRANSACTION_TYPE_CONTRACTS]: statusWriteContract,
-        };
-    return transactionStatuses[transactionType];
-  }, [
-    statusSendCalls,
-    statusWriteContracts,
-    statusSendCall,
-    statusWriteContract,
-    transactionType,
-    walletCapabilities[Capabilities.AtomicBatch],
-  ]);
+    if (walletCapabilities[Capabilities.AtomicBatch]?.supported) {
+      return statusSendCalls;
+    }
+    return statusSendCall;
+  }, [statusSendCall, statusSendCalls, walletCapabilities]);
 
   // Transaction hash for single transaction (non-batched)
-  const singleTransactionHash =
-    writeContractTransactionHash || sendCallTransactionHash;
+  const singleTransactionHash = sendCallTransactionHash;
```

Comment:
Do we still need this assignment? 

Can we remove this and update `useSendCall` `data` return value to be named `singleTransactionHash`?